### PR TITLE
Enhance edit command

### LINKED ISSUE #83: Edit-command-enhancement
edit-date seems to only accept DD-MM-YYYY, not days like "monday", "sunday" etc
would be best to shorten "edit-description", maybe "edit-desc"? It's quite long.
doesn't edit type of task, if user initially add floating task, then edit-date, it doesnt update to event

### PR DESCRIPTION
**Pull request by Mamtha3005**
This closes #83

### CODE REVIEW

**Comment by Mamtha3005** [comment]
@CS2103AUG2016-W15-C3/integration 
@CS2103AUG2016-W15-C3/testing 
@CS2103AUG2016-W15-C3/code-quality 
Ready for testing

**Comment by turtle96** [comment]
Testing pass

some things to consider:
1. edit-startDate allows later date than endDate i.e. endDate is 10-nov, can edit startDate to 17-nov, should notify user of this
2. same issue as above for edit-startTime
3. floating task cannot edit date?

recommend to have a method/code to check start and end dates/times are correct (start before end) before editing task

File: src/main/java/seedu/taskell/logic/commands/EditDescriptionCommand.java [hunk 1]
```diff
@@ -18,14 +18,17 @@
...
+    public static final String COMMAND_WORD_1 = "edit-desc";
+    public static final String COMMAND_WORD_2 = "edit-name";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD
+    public static final String MESSAGE_USAGE = COMMAND_WORD_1
+            + "/"+ COMMAND_WORD_2
             + ": Edits the description task identified by the index number used in the last task listing.\n"
             + "Parameters: INDEX (must be a positive integer) NEW_DESCRIPTION\n"
-            + "Example: " + COMMAND_WORD + " 1 buy cake ";
+            + "Example: " + COMMAND_WORD_1 + " 1 buy cake\n "
```
Comment (turtle96) [inline src/main/java/seedu/taskell/logic/commands/EditDescriptionCommand.java]: remove extra whitespace after "cake\n"

**Comment by turtle96** [comment]
@Mamtha3005 remember to mark your code for Collate

**Pull request merged**